src/compressed_tensors/quantization/quant_scheme.py

```diff
@@ -75,6 +75,22 @@ def default_scheme(
             output_activations=output_activations,
         )
 
+    def remove_defaults(self):
+        result = {}
+        for field, value in dict(self).items():
+            if field == "targets" and len(value) == 0:
```

Comment:
Why can't we compare directly to the default value? This is inflexible if we want to change the targets to default to a non-empty dict